Dockerfile

```diff
@@ -1,56 +1,129 @@
-FROM trailofbits/polytracker-llvm:bbdb72d136af011db9710a4d44a79e6d5059cc1a
+FROM ubuntu:jammy as base
 
 LABEL org.opencontainers.image.authors="evan.sultanik@trailofbits.com"
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get -y update  && \
-      apt-get install -y \
-            ninja-build                                     \
-            python3-pip                                     \
-            python3.8-dev                                   \
-            libgraphviz-dev                                 \
-            libjpeg-dev                                     \
-            graphviz                                        \
-            vim                                             \
-            file                                            \
-            gdb                                             \
-            libncurses5-dev                                 \
-            apt-transport-https                             \
-            ca-certificates                                 \
-            libstdc++-10-dev
+
+ARG BUILD_TYPE="Release"
+ARG PARALLEL_LINK_JOBS=1
+ARG LLVM_TARGET_NAME=X86
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
+  ninja-build                                         \
+  python3-pip                                         \
+  python3.8-dev                                       \
+  python-is-python3                                   \
+  ca-certificates                                     \
+  libstdc++-10-dev                                    \
+  golang                                              \
+  clang-12                                            \
```

Comment:
Should we still depend on / require Clang 12 if the goal is to move away from old Clang? Or are there plans to update this version-by-version once the removing-polytracker-llvm change lands?